common/client-libs/validator-client/src/client.rs

```diff
@@ -182,6 +182,20 @@ impl<C> Client<C> {
         Ok(self.nymd.get_state_params().await?)
     }
 
+    pub async fn get_total_mix_stake(&self) -> Result<u128, ValidatorClientError>
+    where
+        C: CosmWasmClient + Sync,
+    {
+        Ok(self.nymd.get_total_mix_stake().await?.u128())
+    }
+
+    pub async fn get_total_gt_stake(&self) -> Result<u128, ValidatorClientError>
```

Comment:
I think you should rename it to `gateway_stake` for consistency sake, I don't think we ever refer to gateways by `gt` in any public API